config/tests/testgrids/config_test.go

```diff
@@ -63,47 +64,24 @@ var (
 	}
 )
 
-const (
-	prowPath    = "../../../prow/config.yaml"
-	jobPath     = "../../../config/jobs"
-	defaultPath = "../../../config/testgrids/default.yaml"
-)
+var prowPath = flag.String("prow-config", "../../../prow/config.yaml", "Path to prow config")
```

Comment:
Why do we need/want these default values?

Reply:
These tests are specific to this repository, so having a default value that points to this repository's instance of prow-config and prow-jobs is documentation.

They point to the exact files this test is talking about (for this instance), and they work for `go test` as well as they can, since they're from that local working directory.